tajo-client/src/main/java/org/apache/tajo/client/v2/ClientDelegate.java

```diff
@@ -31,7 +32,7 @@
 
   int executeUpdate(String sql) throws TajoException;
 
-  ResultSet executeSQL(String sql) throws TajoException;
+  ResultSet executeSQL(String sql) throws TajoException, QueryFailedException;
```

Comment:
QueryFailedException extends TajoException, so it seems that it is not necessary.